packages/paste-core/components/form/src/FormControlTwoColumn.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+
+export interface FormControlTwoColumnProps extends Omit<React.ComponentPropsWithRef<'legend'>, 'children'> {
```

Comment:
question: why are we getting props from `legend` not `div`?

Reply:
Oops, will fix